README.md

```diff
@@ -21,6 +21,18 @@ $ npm install --save resin-register-device
 Documentation
 -------------
 
+Resin-Register-Device exports a factory function, which takes a single option: `dataDirectory`. This is required in Node.js, but ignored in the browser.
```

Comment:
Slightly ambiguous IMO: can be read as the `dataDirectory` is the parameter itself (as a string) while it's actually inside of the options object.